test/integration/helpers/html-utils.ts

```diff
@@ -32,114 +33,156 @@ export class HtmlElementLookup {
     this.bpmnQuerySelectors = new BpmnQuerySelectorsForTests(bpmnVisualization.graph.container.id);
   }
 
-  private findSvgElement(bpmnId: string): HTMLElement {
-    const bpmnElements = this.bpmnVisualization.bpmnElementsRegistry.getElementsByIds(bpmnId);
-    return bpmnElements.length == 0 ? undefined : bpmnElements[0].htmlElement;
-  }
-
   expectNoElement(bpmnId: string): void {
     const svgGroupElement = this.findSvgElement(bpmnId);
     expect(svgGroupElement).toBeUndefined();
   }
 
-  expectStartEvent(bpmnId: string, checks?: RequestedChecks): void {
+  // ===========================================================================
+  // EVENTS
+  // ===========================================================================
+
+  private expectEventType(bpmnId: string, bpmnClass: string, checks?: RequestedChecks): void {
     const svgGroupElement = this.findSvgElement(bpmnId);
     expectSvgEvent(svgGroupElement);
-    expectSvgElementClassAttribute(svgGroupElement, computeClassValue('bpmn-start-event', checks?.additionalClasses));
+    expectClassAttribute(svgGroupElement, computeClassValue(bpmnClass, checks?.additionalClasses));
+    this.expectSvgLabel(bpmnId, bpmnClass, checks?.label, checks?.additionalClasses);
```

Comment:
ℹ️ here we still have duplication with expectTaskType, expectLane, expectPool, ...
This will be managed in a future PR.